editor.planx.uk/src/ui/public/ExternalPlanningSiteDialog.tsx

```diff
@@ -53,7 +53,6 @@ const getTitleAndContent = (
 
 interface Props {
   purpose: DialogPurpose;
-  teamSettings?: Partial<TeamSettings>;
```

Comment:
It looks like there's a TS issue related to this change causing the built to fail - 
 - "Find all references" on the component and manually check
 - Run `pnpm check` to see if there's any errors
 
To be fair this second one should be running on a pre-commit hook (using Husky) which should block breaking commits. Husky is a bit awkward in mono-repos so I'll take a look at this again to see if we can improve this.